.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -0,0 +1,118 @@
+name: unit_test_CI
+# runs unit tests on AMD64 machine
+
+on: workflow_dispatch
+
+permissions:
+  contents: read
+
+jobs:
+  Start-Runner:
+    name: Start self-hosted EC2 runner
+    runs-on: ubuntu-latest
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Start EC2 runner
+        id: start-ec2-runner
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: start
+          github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          ec2-image-id:  ${{ secrets.AMD_EC2_IMAGE_ID }}
```

Comment:
Since large model tests use CUDA, we should use the secrets corresponding to the CUDA enabled runner. 

### CUDA AMD64 instance
ec2-image-id: ${{ secrets.CUDA_EC2_IMAGE_ID }}
ec2-instance-type: g4dn.xlarge
subnet-id: ${{ secrets.CUDA_SUBNET_ID }}
security-group-id: ${{ secrets.CUDA_SECURITY_GROUP_ID }}